spanner/testdata/protos/singer.proto

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto2";
+
+package protos;
```

Comment:
Package Name will be used later for parameterised queries,  let's better keep it consistent across the clients.  We can use something like `spanner.examples.music`.

Reply:
Updated package name